cocos/core/assets/category.json

```diff
@@ -4,7 +4,7 @@
         "en": "Asset"
     },
     "description": {
-        "zh": "资源",
-        "en": "Asset"
+        "zh": "资源是游戏中的重要组成部分，Cocos Creator 支持导入大量类型的资源，并允许开发者将该资源运用在游戏中",
+        "en": "Assets are an important part of the game, and Cocos Creator supports importing a large number of types of resources and allows developers to use that resource in the game"
```

Comment:
Assets are crucial for game content, Cocos Creator supports importing various types of assets and allows developers to use these resources in the game logic in runtime